### PR TITLE
Ex8 Counting <20

### PR DESCRIPTION
**Pull request by mgilyeat**
Created a function to assign property names to input.
Then count the number of entries with an age < 20 in the array.

### CODE REVIEW

File: students/8/main.js [hunk 1]
```diff
@@ -39,56 +39,77 @@ const ex6 = () => {
...
 
+
+//Ex 8
+
+const createPerson = (name, age) => {
+    return {
+        fullName: name,
+        age: age,
+        // ...other properties
+    };
```
Comment (mburolla) [inline students/8/main.js:63]: Let's remove this semicolon.

File: students/8/main.js [hunk 2]
```diff
@@ -39,56 +39,77 @@ const ex6 = () => {
...
+
+const countOver20 = (personArray) => {
+    let p1 = createPerson("Alice", 11)
+    let p2 = createPerson("Bob", 22)
+    let p3 = createPerson("Charlie", 33)
+
+    personArray.push(p1)
+    personArray.push(p2)
+    personArray.push(p3)
+    //  return personArray;
```
Comment (mburolla) [inline students/8/main.js:74]: Can we delete this code (`//  return personArray;`)?

File: students/8/main.js (hunk 1 above)
Comment (mgilyeat) [reply students/8/main.js:63]: Done

File: students/8/main.js (hunk 2 above)
Comment (mgilyeat) [reply students/8/main.js]: done

File: students/8/main.js [hunk 3]
```diff
@@ -39,56 +39,77 @@ const ex6 = () => {
...
+
+    let counter = 0;
+    for (let i of personArray) {
+        if (i.age > 20 && typeof i.age == 'number') {
+            counter++;
+        }
+    };
```
[the remark this answers is not in the document]
Comment (mgilyeat) [reply students/8/main.js]: done

**Pull request merged**